pkg/mock/provider-alicloud/alicloud/client/mocks.go

```diff
@@ -6,37 +6,36 @@ package client
 
 import (
 	context "context"
-	reflect "reflect"
-
 	vpc "github.com/aliyun/alibaba-cloud-sdk-go/services/vpc"
 	client "github.com/gardener/gardener-extension-provider-alicloud/pkg/alicloud/client"
 	gomock "github.com/golang/mock/gomock"
+	reflect "reflect"
 )
 
-// MockVPC is a mock of VPC interface.
```

Comment:
I think you missed to run `make install-requirements` before `make generate`.

Reply:
Yes, it seems so, should be ok now.